01_separate_projects/flow_dsl/setup.py

```diff
@@ -1,22 +1,12 @@
 from setuptools import setup, find_packages
 
 setup(name='flow_dsl',
-      version='0.1',
-      description='(example)',
-      url='',
-      author='YOUR NAME',
-      author_email='YOUR.NAME@ADDRESS',
-      license='TODO',
       packages=find_packages(),
       package_data={'': ['*.tx']},
-      install_requires=["textx", "arpeggio", "click", "types_dsl", "data_dsl"],
-      tests_require=[
-          'pytest',
-      ],
-      keywords="parser meta-language meta-model language DSL",
+      install_requires=["textx"],
```

Comment:
Should `flow_dsl` require `data_dsl`?

Reply:
Yes. The same holds for data dsl. I will change this

Reply:
fixed